### PR TITLE
add decision_path to decision

### PR DESCRIPTION
**Pull request by Totoketchup**

### CODE REVIEW

File: src/interpreter_v2.js [hunk 1]
```diff
@@ -87,7 +88,7 @@ function decideRecursion(node, context, configuration, outputType, outputValues)
   }
 
   // Regular node
-  const matchingChild = _.find(
+  const matchingChildIndex = _.findIndex(
```
Comment (cloderic) [inline src/interpreter_v2.js]: you can use `node.children.findIndex` to avoid unecessary lodash

File: src/interpreter_v2.js [hunk 2]
```diff
@@ -120,11 +122,11 @@ function decideRecursion(node, context, configuration, outputType, outputValues)
       // then the returned value corresponds to the subtree weighted output values.
       if (outputType === 'enum' || outputType === 'boolean') {
         // Compute the argmax function on the returned distribution:
-        let argmax 
+        let argmax
```
Comment (cloderic) [inline src/interpreter_v2.js]:
```suggestion
        const argmax
```

File: src/interpreter_v2.js [hunk 3]
```diff
@@ -356,15 +362,15 @@ export function computeMeanDistributions(values, sizes) {
...
+  let multiplyByBranchRatio =
   _.zip(values, sizes)
     .map((zipped) => _.map(zipped[0], (val) => val * zipped[1] / totalSize));
   let sumArrays = _sumArrays(multiplyByBranchRatio);
   return { value: sumArrays, size: totalSize };
 }
 
 function _sumArrays(arrays) {
-  return _.reduce(arrays, (acc_sum, array) => 
+  return _.reduce(arrays, (acc_sum, array) =>
```
Comment (cloderic) [inline src/interpreter_v2.js]:
```suggestion
  return arrays.reduce((acc_sum, array) =>
```

File: src/interpreter_v2.js [hunk 4]
```diff
@@ -374,15 +380,16 @@ function decide(configuration, trees, context) {
   // Convert timezones as integers to the standard +/-hh:mm format
   // This should only happen when no Time() object is passed to the interpreter
   const timezoneProperty = getTimezoneKey(configuration.context);
+  let decide_context = _.cloneDeep(context);
```
Comment (cloderic) [inline src/interpreter_v2.js]: Why is a clone needed here?

File: src/interpreter_v2.js (hunk 3 above)
Comment (Totoketchup) [reply src/interpreter_v2.js]: I didn't modify this actually, it's just my editor that corrected the whitespaces. But I agree it's the same ;)

File: src/interpreter_v2.js (hunk 4 above)
Comment (Totoketchup) [reply src/interpreter_v2.js]: otherwise modifying `decide_context` modifies `context`as well and we want to keep the orginal `context`state

**Pull request merged**